### PR TITLE
Add customizable foreground color to CMatrix

### PR DESCRIPTION
**Pull request by 0xNiffin**
Just wanted to be able to change the color on the CMatrix characters :)

### CODE REVIEW

File: res/config.ini [hunk 1]
```diff
@@ -2,7 +2,7 @@
 # none   -> Nothing (default)
 # doom   -> PSX DOOM fire
 # matrix -> CMatrix
-animation = none
+animation = matrix
```
Comment (AnErrupTion) [inline res/config.ini:5]: Please don't change the default option to use an animation.

File: res/config.ini [hunk 2]
```diff
@@ -47,6 +47,9 @@ fg = 8
 # Border color
 border_fg = 8
 
+#CMatrix foreground color id
```
Comment (AnErrupTion) [inline res/config.ini:50]: Please add a space after the `#` for consistency. :)

File: res/config.ini [hunk 3]
```diff
@@ -164,4 +167,4 @@ xsessions = /usr/share/xsessions
 # Brightness control
 brightness_down_key = F5
 brightness_up_key = F6
-Brightness_change = 10
\ No newline at end of file
+Brightness_change = 10
```
Comment (AnErrupTion) [inline res/config.ini:170]: I didn't see this beforehand but please use a lowercase `b` here.

File: src/animations/Matrix.zig [hunk 4]
```diff
@@ -66,7 +66,7 @@ pub fn realloc(self: *Matrix) !void {
     self.lines = lines;
 }
 
-pub fn draw(self: *Matrix) void {
+pub fn draw(self: *Matrix, fg_ini: u16) void {
```
Comment (AnErrupTion) [inline src/animations/Matrix.zig:69]: Please add `fg_ini` as a field in `Matrix` instead.

File: src/animations/Matrix.zig [hunk 5]
```diff
@@ -145,7 +145,8 @@ pub fn draw(self: *Matrix) void {
         var y: u64 = 1;
         while (y <= self.terminal_buffer.height) : (y += 1) {
             const dot = self.dots[buf_width * y + x];
-            var fg: u16 = @intCast(termbox.TB_GREEN);
+            
+            var fg: u16 = fg_ini;
```
Comment (AnErrupTion) [inline src/animations/Matrix.zig:149]: You can also get rid of this variable.

**Review by AnErrupTion — CHANGES_REQUESTED** [review]

**Comment by 0xNiffin** [comment]
Sorry it's my first time dealing with something in Zig haha

I've made a commit with the new changes, for some weird reason it's not letting me change the `var fg: u16 = @intCast(termbox.TB_GREEN);` as it is not present the code.

Could you please elaborate on how I could add that variable as a field? I've looked at the documentation but left confused.

Thanks in advance! Cheers

**Comment by 0xNiffin** [comment]
Hello again!

Did the changes your requested, I've also noticed that the capital B (res/config.ini) is actually present in the main branch of ly.

Cheers

**Comment by AnErrupTion** [comment]
> I've also noticed that the capital B (res/config.ini) is actually present in the main branch of ly.

Yup, I simply didn't notice that when merging the PR that had these changes. Other than that, the changes look good, thanks!

**Review by AnErrupTion — APPROVED** [review]

**Pull request merged**